src/Stratis.Bitcoin.Tests/P2P/PeerSelectorTests.cs

```diff
@@ -836,7 +836,36 @@ public void PeerSelector_ReturnConnectedPeers_AfterHandshakeFailure_HandshakeSuc
             Assert.Equal(0, peer.HandshakedAttempts);
             Assert.Null(peer.LastHandshakeAttempt);
         }
-        
+
+        [Fact]
+        public void PeerSelector_ReturnConnectedPeers_AfterThresholdExpired_ResetCounters()
```

Comment:
I don't understand this test title, can you make it clearer?

Perhaps a fix as simple as 
`....._Reset*s*Counters`

If that was the intention, the side effect being tested is that the counters get reset ?

Reply:
@thefazzer I've fixed these for you.